.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,58 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+  push:
+    tags:
+      - 'v.[0-9]+.[0-100]+.[0-100]+'
```

Comment:
Please follow this format https://github.com/h2oai/mojo2/blob/974ae7904ae3f2c85a5d16b52ed788f14d361908/.github/workflows/snyk-scan.yml#L4 as mentioned in other PRs

Reply:
sure